app/routes/companyInterest/CompanyInterestListRoute.ts

```diff
@@ -40,18 +45,32 @@ const mapStateToProps = (state, props) => {
           })
         : 'Vis alle semestre',
   };
+  const selectedEventOption = {
+    value: eventValue ? eventValue : '',
+    label: eventValue
+      ? EVENT_TYPE_OPTIONS.filter(
+          (eventType) => eventType.value === eventValue
+        ).at(0).label
+      : 'Vis alle arrangementstyper',
+  };
   const companyInterestList = selectCompanyInterestList(
     state,
-    selectedOption.id
+    selectedSemesterOption.id,
+    selectedEventOption.value
   );
   const hasMore = state.companyInterest.hasMore;
   const fetching = state.companyInterest.fetching;
+  const event_string = selectedEventOption.value
```

Comment:
Also knitpickery, but everything else here uses camelCase, is there a reason this uses snake_case?

Reply:
I deleted this one. But I would agree 👍